### PR TITLE
Update 096-message-tributes.md

### PR DESCRIPTION
**Pull request by rachelhamlin**
Add contributors, update status.

### CODE REVIEW

File: ideas/096-message-tributes.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 id: 096-message-tributes
 title: Tribute to Talk
-status: Limbo
```
Comment (oskarth) [inline ideas/096-message-tributes.md]: Limbo is a keyword - maybe we can keep it in Limbo and leave a comment like you have below? Alt we could introduce a new keyword, but seems a bit messy

Comment (rachelhamlin) [reply ideas/096-message-tributes.md]: Didn't realize—keep as Limbo!

Comment (oskarth) [reply ideas/096-message-tributes.md]: @rachelhamlin do you want to fix and we can merge this?

Comment (rachelhamlin) [reply ideas/096-message-tributes.md]: @oskarth fixed :)

**Pull request merged**